pkgs/applications/accessibility/svkbd/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, xlibs
+, layout ? null
```

Comment:
I guess this is problematic in case there's ever a package called `layout`?

Reply:
Yeah, that would cause issues.

Reply:
I could add `lib.isString layout` do prevent that.

Reply:
But then again, that would fail quietly in some cases, so I think it should be left as is.